### PR TITLE
ApeStake: ApeCoin Pool Apy

### PR DESCRIPTION
**Pull request by tommy4241**

### CODE REVIEW

File: data/mainnet/tokens.js [hunk 1]
```diff
@@ -45,6 +45,41 @@ module.exports = {
...
+    },
+  },
+  apeStake: {
+    isNew: true,
+    category: VAULT_CATEGORIES_IDS.APESTAKE,
+    chain: CHAINS_ID.ETH_MAINNET,
+    logoUrl: './icons/apecoin.png',
+    apyIconUrls: ['./icons/apecoin.png'],
+    apyTokenSymbols: ['APE'],
+    displayName: 'ApeStake',
```
Comment (CryptJS13) [inline data/mainnet/tokens.js:66]: Lets rename to 'APE'

File: data/mainnet/tokens.js [hunk 2]
```diff
@@ -45,6 +45,41 @@ module.exports = {
...
+    subLabel: 'ApeStake APECOIN Pool',
```
Comment (CryptJS13) [inline data/mainnet/tokens.js:67]: This line can be removed

**Review by CryptJS13 — CHANGES_REQUESTED** [review]
Hey @tommy4241, few comments that need to be fixed. Thanks!

File: src/vaults/apys/implementations/ape-stake.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    apeCoinStakingContract.abi,
+    apeCoinStakingContract.address.mainnet,
+  )
+
+  const apePool = await getPool(APE_POOL_ID, apeCoinStakingInstance)
+  const currentTimeStamp = Math.floor(Date.now() / 1000)
+  const rewardInfo = await getRewardsBy(
+    APE_POOL_ID,
+    currentTimeStamp,
+    currentTimeStamp + ONE_YEAR,
```
[the remark this answers is not in the document]
Comment (tommy4241) [reply src/vaults/apys/implementations/ape-stake.js:21]: I see. Good Point!

**Review by CryptJS13 — APPROVED** [review]
Thanks! Looks good!

**Pull request merged**